src/main/java/com/google/sps/data/WorkingHours.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.sps.data;
+
+public class WorkingHours {
+
+  private final int startHour;
+  private final int startMin;
+  private final int endHour;
+  private final int endMin;
```

Comment:
This works and is ok for now, given the time left. I would however recommend instead storing the hour and minute together in a [`java.time.LocalTime`](https://docs.oracle.com/javase/8/docs/api/java/time/LocalTime.html) which could then be used for assertions (`isBefore` and `isAfter` checks) and also converted into the format you need in the scheduler.